appcenter-analytics/Analytics.d.ts

```diff
@@ -0,0 +1,6 @@
+export function trackEvent(
+  eventName: string,
+  properties?: { [name: string]: string | number | boolean }
```

Comment:
We support only string property values

Reply:
I thought so but then looked at the `sanitizeProperties` function and saw it parse strings, numbers and booleans too I think?

Reply:
We have custom properties for push which are typed and not using a dictionary anyway to avoid errors.

Custom properties and event properties are 2 different features. Event properties are just string values.

Reply:
@guperrot This confusion caused by [this conversion to string](https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/appcenter-analytics/Analytics.js#L31-L35). And if it already here I guess it's OK.

Reply:
The only reason we have this is because object was used to map strings because javascript is not strongly typed. But in all typed SDKs we have it's really string values that we are allowing and sending a number would be converted to a string anyway. Exposing more types will be misleading.

Reply:
👍 Should be fixed. This one does return a `string | null`? There seems a lot more opportunity for errors here, but the docs mention same as with `isEnabled` (`This function returns a promise, in which the success case returns an empty string, and the error case returns the error.`)